yarn.lock

```diff
@@ -5187,7 +5189,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"axios@npm:^1.7.4, axios@npm:^1.7.9":
+"axios@npm:^1.7.4, axios@npm:^1.7.7":
```

Comment:
I'm not sure if this is supposed to be this way

please run `yarn` to re-sync your packages